script/Deploy.s.sol

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+// Copyright (C) 2024 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity ^0.8.21;
+
+import "forge-std/Script.sol";
+
+import { ScriptTools } from "dss-test/ScriptTools.sol";
+import { Domain } from "dss-test/domains/Domain.sol";
+import { TokenBridgeDeploy, L1TokenBridgeInstance, L2TokenBridgeInstance } from "deploy/TokenBridgeDeploy.sol";
+import { ChainLog } from "deploy/mocks/ChainLog.sol";
+import { GemMock } from "test/mocks/GemMock.sol";
+
+// TODO: Add to dss-test/ScriptTools.sol
+library ScriptToolsExtended {
+    VmSafe private constant vm = VmSafe(address(uint160(uint256(keccak256("hevm cheat code")))));
+    function exportContracts(string memory name, string memory label, address[] memory addr) internal {
+        name = vm.envOr("FOUNDRY_EXPORTS_NAME", name);
+        string memory json = vm.serializeAddress(ScriptTools.EXPORT_JSON_KEY, label, addr);
```

Comment:
This fixed the issue of not losing the other contracts in the output for me:
```suggestion
        string memory json = vm.serializeAddress(string(abi.encodePacked(ScriptTools.EXPORT_JSON_KEY, "_", name)), label, addr);
```

I think we should upgrade `dss-test` to the last version in the other 3 repos as well and apply the patch. Or maybe just better to add this to `dss-test` and upgrade it everywhere.

Reply:
Sorry, hadn't seen your suggestion, but I applied the same fix.

Reply:
Added the functions to dss-test https://github.com/makerdao/dss-test/pull/46

Reply:
Great! will check the PR.